test-utils/src/base-fake-repository.ts

```diff
@@ -0,0 +1,72 @@
+const { validate } = require('class-validator');
+
+export class BaseFakeRepository {
+  #__rows: Array<any>;
+  #__ModelClass: any;
+
+  constructor(ModelClass: any) {
+    this.#__rows = [];
```

Comment:
Move `= []` to the field on the top of the constructor.

Reply:
fixed